src/appleseed/foundation/math/population.h

```diff
@@ -56,8 +56,8 @@ class Population
     // Constructor.
     Population();                           // empty population
 
-    // Insert a new value into the population.
-    void insert(const ValueType& val);
+    // Insert count times the value val into the population.
```

Comment:
@gospodnetic started using backticks to reference parameters in comments and since then it has become somewhat of a convention, e.g.

    // Insert `count` times the value `val` into the population.

Helps readability :)